### ISSUE TITLE
[Mac] Download Keyboard in Configuration shows nothing

### ISSUE THREAD

**Issue by rfaramir** [keymanapp/keyman#1618]
**Describe the bug**
The "Download Keyboard..." dialog shows nothing. Just a blank white pane.

**To Reproduce**
1. Activate Keyman and select Keyman Configuration from the input methods menu. 
2. Click **Download keyboard**.
3. It shows a blank white pane.

**Expected behavior**
It should show the download page or the Search box.

**Screenshots**
![screen shot 2019-02-22 at 3 10 05 pm](https://user-images.githubusercontent.com/45110266/53271411-035dc180-36b4-11e9-86c5-354105721130.png)

---

**Keyman Desktop, Keyman for macOS, Keyman for Linux (please complete the following information):**
 - OS: macOS Mojave
 - Keyman version: 11.0 (just before stable)

**Comment by darcywong00** [comment]
Is that window supposed to open the following link?

KMDownloadKBWindowController.m line 30:
```
    NSString *url = [NSString stringWithFormat:@"https://keyman.com/go/macos/10.0/download-keyboards/?version=%@", version];
```

Maybe verify you can go to:
https://keyman.com/go/macos/10.0/download-keyboards/?version=11.0

btw, I've staged a redirect in sillsdev/keyman-keyman.com#246 for
https://keyman.com/go/macos/11.0/download-keyboards/?version=11.0

**Comment by mcdurdin** [comment]
This appears to be already resolved. Working for me?

**Comment by mcdurdin** [comment]
Fixed by #1814.
